girder/api/v1/user.py

```diff
@@ -235,11 +235,11 @@ def updateUser(self, user, firstName, lastName, email, admin, status):
         user['email'] = email
 
         # Only admins can change admin state
-        if admin is True:
+        if admin in {True, False}:
             if self.getCurrentUser()['admin']:
                 user['admin'] = admin
-            elif not user['admin']:
-                raise AccessException('Only admins may enable admin state.')
+            elif user['admin'] != admin:
```

Comment:
Could this always raise the exception if the `admin` parameter is provided but the current user is not an admin?